assets/airports/essa.json

```diff
@@ -2660,7 +2662,83 @@
             "draw": [
                 ["XILAN", "SA619", "SA618", "SA604", "SA605", "SA802", "SA801"]
             ]
-        }
+        },
+        "ELTOK7M": {
+            "icao": "ELTOK7M",
+            "name": "Eltok Seven Mike",
+            "holds":{
+                "ELTOK": "286|left|19min|S230-",
```

Comment:
Throughout the STAR holds, 19min is a placeholder for decimal values that will be specified when #1742  is merged.

Reply:
Fixed in later commit.